sound/soc/intel/boards/sof_sdw_rt715.c

```diff
@@ -18,4 +31,7 @@ void sof_sdw_rt715_init(const struct snd_soc_acpi_link_adr *link,
 	 */
 	if (sof_sdw_quirk & SOF_RT715_DAI_ID_FIX)
 		dai_links->id = SDW_DMIC_DAI_ID;
+
+	dai_links->init = rt715_rtd_init;
+
```

Comment:
this new line doesn't seem to be very useful either

Reply:
same as in other files, so keeping the same style

Reply:
strange, as I was writing this comment I was commenting a case of an empty line before a closing curly brace. Like
```
if (x) {
	line1();
	line2();

}
```
Now there isn't such a pattern here. Either I have misplaced the comment or GH has. Anyway, sorry in case it was my mistake.